create-dev-testfile.sh

```diff
@@ -31,7 +31,11 @@ terraform {
   }
 }
 
-provider "prefect" {}
+provider "prefect" {
+$([ -n "$PREFECT_API_URL" ] && echo '  endpoint = "'$PREFECT_API_URL'"')
+$([ -n "$PREFECT_API_KEY" ] && echo '  api_key = "'$PREFECT_API_KEY'"')
+$([ -n "$PREFECT_CLOUD_ACCOUNT_ID" ] && echo '  account_id = "'$PREFECT_CLOUD_ACCOUNT_ID'"')
+}
```

Comment:
so if these env vars are set, then running `make dev-new ...` will auto-populate the generated provider block (i got tired of having to copy paste every time)

Reply:
@parkedwards - as an alternative, I recently added [a script](https://github.com/PrefectHQ/terraform-provider-prefect/blob/main/scripts/testacc-dev) to facilitate this by pulling the values from 1Password in case that fits into your workflow (I ran into the same thing)

Reply:
Mm, I think your approach helps when we're _not_ testing against the `github-ci-tests` workspace. I'd need to update my script to let `vault_entry` be changed to another entry.

Reply:
right, this was just for the `make dev-new` script that generates a local (gitignored) TF dir for some manual testing.  totally an optional facility, the acceptance testing suite should be the source of truth